app/usage.go

```diff
@@ -19,3 +20,13 @@ func (a *App) GetPostsUsage() (int64, *model.AppError) {
 
 	return utils.RoundOffToZeroes(float64(count)), nil
 }
+
+// GetStorageUsage returns the sum of files stored
+func (a *App) GetStorageUsage() (int64, *model.AppError) {
```

Comment:
Missing test.

Reply:
I left it out because I don't think it adds anything valuable. The test would essentially be doing the same exact thing as the store test.